waspc/examples/crud-testing/src/DetailPage.tsx

```diff
@@ -1,15 +1,20 @@
-import "./Main.css";
-import React from "react";
-import { useParams } from "react-router-dom";
-import { Link } from "wasp/client/router";
+import './Main.css'
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import { Link } from 'wasp/client/router'
 
-import { tasks as tasksCrud } from "wasp/client/crud";
+import { tasks as tasksCrud } from 'wasp/client/crud'
 
 const DetailPage = () => {
-  const { id } = useParams<{ id: string }>();
-  const { data: task, isLoading } = tasksCrud.get.useQuery({
-    id: parseInt(id, 10),
-  });
+  const { id } = useParams<{ id: string }>()
```

Comment:
Not related to this PR, but a quick fix since `id` is of type `string | undefined`.